profitbricks-rest/src/test/java/org/apache/jclouds/profitbricks/rest/internal/BaseProfitBricksApiMockTest.java

```diff
@@ -89,13 +89,13 @@ protected MockResponse response204() {
    protected MockResponse response404() {
```

Comment:
No need for this then. :) Also, remove 404 tests (or Unauthorized, if exists) from other tests, if possible.